locales/en-US/server.ftl

```diff
@@ -315,6 +315,7 @@ shotIndexFavoriteIcon =
     .title = This is a favorite shot and it does not expire
 shotIndexSyncedShot =
     .title = Shot taken on another device
+shotIndexAlertErrorFavoriteShot = Error updating favorite shot status
```

Comment:
Not sure about this message.  Perhaps @johngruen can advise.  It's similar to [the one on the shot page](https://github.com/mozilla-services/screenshots/blob/master/locales/en-US/server.ftl#L93).  (That one probably should be updated?)  It's used for favorite _and_ un-favorite actions.